DQM/L1TMonitor/src/L1TObjectsTiming.cc

```diff
@@ -0,0 +1,480 @@
+    etsum_eta_phi_MHTHF.at(i)->setAxisTitle("#phi", 1);
+  }
+
+
+  ibooker.setCurrentFolder(monitorDir+"/L1TMuon"+"/timing"+"/Isolated_bunch");  
+  for(unsigned int i=0; i<bxrange; ++i) {
+    muons_eta_phi_isolated.push_back(ibooker.book2D("muons_eta_phi_bx_isolated_"+bx_obj[i],"L1T Muon #eta vs #phi for isolated bunch BX="+bx_obj[i],25, -2.5, 2.5, 25, -3.2, 3.2));
+    muons_eta_phi_isolated.at(i)->setAxisTitle("#eta", 1);
+    muons_eta_phi_isolated.at(i)->setAxisTitle("#phi", 2);
+  }
+  
+  ibooker.setCurrentFolder(monitorDir+"/L1TJet"+"/timing"+"/Isolated_bunch");
+  for(unsigned int i=0; i<bxrange; ++i) {
+    jet_eta_phi_isolated.push_back(ibooker.book2D("jet_eta_phi_bx_isolated_"+bx_obj[i],"L1T Jet #eta vs #phi for isolated bunch BX="+bx_obj[i],25, -2.5, 2.5, 25, -3.2, 3.2));
+    jet_eta_phi_isolated.at(i)->setAxisTitle("#eta", 1);
+    jet_eta_phi_isolated.at(i)->setAxisTitle("#phi", 2);
+  }
+
+  ibooker.setCurrentFolder(monitorDir+"/L1TEGamma"+"/timing"+"/Isolated_bunch");
+  for(unsigned int i=0; i<bxrange; ++i) {
+    egamma_eta_phi_isolated.push_back(ibooker.book2D("egamma_eta_phi_bx_isolated_"+bx_obj[i],"L1T EGamma #eta vs #phi for isolated bunch BX="+bx_obj[i],25, -2.5, 2.5, 25, -3.2, 3.2));
+    egamma_eta_phi_isolated.at(i)->setAxisTitle("#eta", 1);
+    egamma_eta_phi_isolated.at(i)->setAxisTitle("#phi", 2);
+  }
+
+  ibooker.setCurrentFolder(monitorDir+"/L1TTau"+"/timing"+"/Isolated_bunch");
+  for(unsigned int i=0; i<bxrange; ++i) {
+    tau_eta_phi_isolated.push_back(ibooker.book2D("tau_eta_phi_bx_isolated_"+bx_obj[i],"L1T Tau #eta vs #phi for isolated bunch BX="+bx_obj[i],25, -2.5, 2.5, 25, -3.2, 3.2));
+    tau_eta_phi_isolated.at(i)->setAxisTitle("#eta", 1);
+    tau_eta_phi_isolated.at(i)->setAxisTitle("#phi", 2);
+  }
+
+  ibooker.setCurrentFolder(monitorDir+"/L1TEtSum"+"/timing"+"/Isolated_bunch");
+  for(unsigned int i=0; i<bxrange; ++i) {
+    etsum_eta_phi_MET_isolated.push_back(ibooker.book1D("etsum_phi_bx_MET_isolated_"+bx_obj[i],"L1T MET #phi for isolated bunch BX="+bx_obj[i],25, -3.2, 3.2));
+    etsum_eta_phi_MET_isolated.at(i)->setAxisTitle("#phi", 1);
+    etsum_eta_phi_METHF_isolated.push_back(ibooker.book1D("etsum_phi_bx_METHF_isolated_"+bx_obj[i],"L1T METHF #phi for isolated bunch BX="+bx_obj[i],25, -3.2, 3.2));
+    etsum_eta_phi_METHF_isolated.at(i)->setAxisTitle("#phi", 1);
+    etsum_eta_phi_MHT_isolated.push_back(ibooker.book1D("etsum_phi_bx_MHT_isolated_"+bx_obj[i],"L1T MHT #phi for isolated bunch BX="+bx_obj[i],25, -3.2, 3.2));
+    etsum_eta_phi_MHT_isolated.at(i)->setAxisTitle("#phi", 1);
+    etsum_eta_phi_MHTHF_isolated.push_back(ibooker.book1D("etsum_phi_bx_MHTHF_isolated_"+bx_obj[i],"L1T MHTHF #phi for isolated bunch BX="+bx_obj[i],25, -3.2, 3.2));
+    etsum_eta_phi_MHTHF_isolated.at(i)->setAxisTitle("#phi", 1);
+  }
+
+  
+  ibooker.setCurrentFolder(monitorDir+"/L1TMuon"+"/timing"+"/First_bunch");
```

Comment:
Only create and fill these MEs if the first bunch trigger bit is was found. So `algoBitFirstBxInTrain_ > -1`